proto/minder/v1/minder.proto

```diff
@@ -459,6 +514,7 @@ service ProfileService {
 
         option (rpc_options) = {
             auth_scope: OBJECT_OWNER_PROJECT
+            relation: RELATION_PROFILE_GET
```

Comment:
same comment as below. A dedicated relation for `profile_status_get` would be ideal.